### PR TITLE
feat: add open graph tags

### LINKED ISSUE #324: Add Open Graph tags to pages
## Description
Add title, description and thumbnail image tags to each page.

### PR DESCRIPTION
**Pull request by paulschreiber**
## Description
Adds open graph title, description and image tags and <meta> description tags.

### Checklist
- [X] Corresponding issue has been opened
- [ ] New tests added
- [x] English strings reviewed and copyedited
- [x] Strings are localized

### Related Issues
Fixes #324, Fixes https://github.com/techmatters/terraso-product/issues/9
Backend https://github.com/techmatters/terraso-backend/pull/423

### CODE REVIEW

**Comment by paulschreiber** [comment]
@josebui can you translate the strings?

**Review by JeancarloBarrios — APPROVED** [review]
LGTM

**Review by david-code — COMMENTED** [review]
Looks good! This seems to be a good way to handle this problem.

File: package-lock.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "terraso-web-client",
   "version": "0.1.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 3,
```
[the remark this answers is not in the document]
Comment (paulschreiber) [reply package-lock.json:4]: not sure.

File: package.json [hunk 2]
```diff
@@ -37,6 +37,7 @@
     "react-dom": "^18.2.0",
     "react-dropzone": "^14.2.3",
     "react-error-boundary": "^3.1.4",
+    "react-helmet-async": "^1.3.0",
```
[the remark this answers is not in the document]
Comment (paulschreiber) [reply package.json:40]: This is definitely used. That's what's allowing us to add these metadata tags.

**Pull request merged**